src/fragments/lib-v1/storage/js/getting-started.mdx

```diff
@@ -68,6 +68,11 @@ Amplify.configure({
     S3: {
       bucket: 'XXX', //REQUIRED -  Amazon S3 bucket name
       region: 'XX-XXXX-X', //OPTIONAL -  Amazon service region
+    }
+  }
+}, {
+  Storage: {
+    S3: {
```

Comment:
Should `isObjectLockEnabled: true` be in above `Storage` block why do we have 2 separate S3 configs here?

Reply:
This is correct, object lock enabled status is toggled with Library options, which is the 2nd param to configure